test/csx.coffee

```diff
@@ -0,0 +1,616 @@
+# We usually do not check the actual JS output from the compiler, but since
+# CSX is not readily supported by Node, we do it in this case
+eqCSX = (cs, js) ->
```

Comment:
@GeoffreyBooth is that boilerplate useful? I find the tests easier to read without it.